browser/scripts/ui-site.js

```diff
@@ -84,7 +87,7 @@ var siteUI = new function() {
 		VizorUI.enablePopupEmbedLinks($body);
 
 		var signupCallback = function() {	// slush
-			window.location.href = '/edit';
```

Comment:
@kschzt this edge-case fallback call takes effect only if you manage to somehow hit /signup, which you can if you hit /login but the paths to /login are blocked by modal windows that go to /edit, except in the case of /forgot. The idea for the next stage of this build is to go to /%user% rather than /edit, from all places except the editor, this is yet to be completed.